pymess/backend/push/onesignal.py

```diff
@@ -0,0 +1,56 @@
+from http import HTTPStatus
+from json.decoder import JSONDecodeError
+
+import requests
+from django.utils import timezone
+from django.utils.encoding import force_text
+from onesignal import DeviceNotification, OneSignal
+
+from pymess.backend.push import PushNotificationBackend
+from pymess.config import settings
+from pymess.models import PushNotificationMessage
+from pymess.utils.logged_requests import generate_session
+
+
+class OneSignalPushNotificationBackend(PushNotificationBackend):
+
+    def _is_response_error(self, response):
+        return response.status_code != HTTPStatus.OK
+
+    def _is_response_partial_error(self, response):
+        return not self._is_response_error(response) and response.json().get('errors')
```

Comment:
pozor server taky nemusí vrátit json když vrátí jiný status kód než 500 bude to pak padat nad response.json

Reply:
Máš pravdu, ale vycházel jsem z dokumentace OneSignal. Tenhle call by měl vždycky vrátit JSON. https://documentation.onesignal.com/reference#section-results-create-notification

Radši to ale opravím, nebudeme na to spoléhat.